pyvista/core/grid.py

```diff
@@ -101,7 +121,7 @@ class RectilinearGrid(_vtk.vtkRectilinearGrid, Grid):
 
     _WRITERS = {'.vtk': _vtk.vtkRectilinearGridWriter, '.vtr': _vtk.vtkXMLRectilinearGridWriter}
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, check_duplicates=False, **kwargs):
```

Comment:
It seems we're not using these `**kwargs` anywhere. Should we pass them on to `_from_file` like we do in `StructuredGrid.__init__()`?

Reply:
I updated this so that kwargs are passed to `_from_file`. 

Generally, I do not like what is going on there and think another PR to clean this up is needed so that if a user passes unused kwargs it gets handled